src/services/meta.ts

```diff
@@ -9,13 +9,29 @@ const META_URL = "https://meta.yearn.network";
  */
 export class MetaService extends Service {
   async tokens(addresses?: Address[]): Promise<TokenMetadata[]> {
-    const tokensMetadata = await fetch(`${META_URL}/tokens/${this.chainId}/all`).then((res) => res.json());
+    const tokensMetadata: TokenMetadata[] = await fetch(`${META_URL}/tokens/${this.chainId}/all`).then((res) =>
```

Comment:
nit: It would be nice to make `fetch()` generic, so that we could `fetch<TokenMetadata[]>()`

No action needed, just a thought.

Reply:
agree on this, we could have a wrapper or define a type for it